markdown/pxf/HDFSWritablePXF.html.md.erb

```diff
@@ -396,7 +394,7 @@ The field type of `recordkey` must correspond to the key type, much as the other
 
 ### <a id="example1"></a>Example: Using Record Keys
 
-Create an external readable table to access the record keys from the writable table `pxf_tbl_seqwrit` that you created in [Example: Writing Data Using the SequenceWritable Profile](#profile_hdfsseqwritable_writing).  The `recordkey` is of type `int8`. (We did not identify a record key when creating the writable table.
+Create an external readable table to access the record keys from the writable table `pxf_tbl_seqwrit` that you created in [Example: Writing Data Using the SequenceWritable Profile](#profile_hdfsseqwritable_writing).  The `recordkey` is of type `int8`. (You did not identify a record key when you created the writable table.
```

Comment:
those two sentences are not directly related.  the type of recordkey will always be int8.  i am going to just remove the "You did not identify a record key..." sentence as a variant is used after the example.